docs/source/user_guides/context_guide.rst

```diff
@@ -35,7 +35,7 @@ Let's consider some of the built-in callback instances to see how the context ca
       pattern = re.compile("[a-zA-Z]+")
 
       def regexp_condition_handler(
-          ctx: Context, pipeline: Pipeline, *args, **kwargs
+          ctx: Context, pipeline: Pipeline
       ) -> bool:
```

Comment:
Collapse to single line, maybe?

Reply:
Done in 48b69340ef8aaf0c7a229a47c7ee352d61f706cc.